routes/html/index.js

```diff
@@ -1,20 +1,27 @@
 const Users = require("../../models/Users");
 const Games = require("../../models/Games");
 const sequelize = require("../../config/connection");
+const { withAuth } = require("../../utils/auth");
 
 const router = require("express").Router();
 
-router.get("/", (req, res) => {
+router.get("/", withAuth, async (req, res) => {
   const loggedIn = req.user;
-  if (loggedIn) {
+  if (!!loggedIn) {
     // render the dashboard
-    res.render("dashboard", { loggedIn });
+    const fetchName = req.user.firstName;
+    const fetchUsername = req.user.username;
+    res.render("dashboard", { fetchName, fetchUsername, loggedIn });
   } else {
     // render the landing page
     res.render("landingpage");
   }
 });
 
+router.get("/landing", (req, res) => {
+  res.render("landingpage");
+});
+
 router.get("/login", (req, res) => {
   const loggedIn = req.user;
```

Comment:
same as above. if you copy code blindly, you copy along the mistakes...

Reply:
fixed in other PR